packages/@uppy/audio/src/index.js

```diff
@@ -0,0 +1,403 @@
+const { h } = require('preact')
+const { UIPlugin } = require('@uppy/core')
+const getFileTypeExtension = require('@uppy/utils/lib/getFileTypeExtension')
+const supportsMediaRecorder = require('./supportsMediaRecorder')
+const RecordingScreen = require('./RecordingScreen')
+const PermissionsScreen = require('./PermissionsScreen')
+const locale = require('./locale.js')
+const packageJsonVersion = require('../package.json').version
```

Comment:
I don't think we can do that, otherwise the string is not replaced by the actual version at build time. See https://github.com/andrewimm/babel-plugin-inline-package-json#readme.